procmon/processmonitor.cpp

```diff
@@ -6568,33 +6497,26 @@ int ProcessMonitor::glusterAssign(std::string dbrootID)
         moduleIPAddr = sysConfig->getConfig("SystemModuleConfig", dataDupIPaddr);
     }
 
-    if ( rootUser)
-    {
-        command = "mount -tglusterfs -odirect-io-mode=enable " + moduleIPAddr + ":/dbroot" +
-                  dbrootID + " " + startup::StartUp::installDir() + "/data" + dbrootID + " > /tmp/glusterAssign.txt 2>&1";
-    }
-    else
-    {
-        command = "sudo mount -tglusterfs -odirect-io-mode=enable " + moduleIPAddr + ":/dbroot" +
-                  dbrootID + " " + startup::StartUp::installDir() + "/data" + dbrootID + " > /tmp/glusterAssign.txt 2>&1";
-    }
+	string tmpLog = tmpLogDir + "/glusterAssign.log";
+    command = "mount -tglusterfs -odirect-io-mode=enable " + moduleIPAddr + ":/dbroot" + dbrootID + " " + startup::StartUp::installDir() + "/data" + dbrootID + " > " + tmpLog + " 2>&1";
 
```

Comment:
lots of mounting and unmounting going on above, none if it will work as not-root.